aspnetcore/zone-pivot-groups.yml

```
@@ -85,7 +85,9 @@ groups:
   prompt: Choose the app specification
   pivots:
     - id: without-bff-pattern
-      title: OIDC without BFF pattern
+      title: OIDC without BFF pattern (Auto)
+    - id: without-bff-pattern-server
+      title: OIDC without BFF pattern (Server)
     - id: with-bff-pattern
       title: OIDC with BFF pattern
```

Should we mention that this is (Auto) too?

```suggestion
      title: OIDC with BFF pattern (Auto)
```